compatibility_lib/compatibility_lib/compatibility_store.py

```diff
@@ -126,63 +126,63 @@ def timestamp(self, timestamp: datetime.datetime):
 class CompatibilityStore:
     """Storage for package compatibility information."""
 
-    def __init__(self, project_id=None):
-        self._client = bigquery.Client(project=project_id)
-        dataset_ref = self._client.dataset(_DATASET_NAME)
-
-        self._self_table_id = (
-            '{}.{}'.format(
-                _DATASET_NAME, _SELF_COMPATIBILITY_STATUS_TABLE_NAME))
-        self._self_table = self._client.get_table(dataset_ref.table(
-            _SELF_COMPATIBILITY_STATUS_TABLE_NAME))
-
-        self._pairwise_table_id = (
-            '{}.{}'.format(
-                _DATASET_NAME, _PAIRWISE_COMPATIBILITY_STATUS_TABLE_NAME))
-        self._pairwise_table = self._client.get_table(dataset_ref.table(
-            _PAIRWISE_COMPATIBILITY_STATUS_TABLE_NAME))
-
-        self._release_time_table_id = (
-            '{}.{}'.format(
-                _DATASET_NAME, _RELEASE_TIME_FOR_DEPENDENCIES_TABLE_NAME))
-        self._release_time_table = self._client.get_table(dataset_ref.table(
-            _RELEASE_TIME_FOR_DEPENDENCIES_TABLE_NAME))
+    def __init__(self, mysql_user=None, mysql_password=None):
+        if mysql_user is None:
+            mysql_user = os.environ.get('MYSQL_USER')
+        if mysql_password is None:
+            mysql_password = os.environ.get('MYSQL_PASSWORD')
+
+        self.mysql_user = mysql_user
+        self.mysql_password = mysql_password
+
+    def connect(self):
+        # Assumes that the database is running local or we are connecting to
```

Comment:
```suggestion
        # Assumes that the database is running locally or we are connecting to
```

Sorry, my original grammar was bad ;-)

Reply:
Done.